src/hooks/screen/useChatScreenHook.ts

```diff
@@ -6,6 +6,7 @@ import SimpleToast from 'react-native-simple-toast';
 import {v4 as uuid} from 'uuid';
 
 import AnonymousMessageRepo from '../../service/repo/anonymousMessageRepo';
+import ChannelList from '../../database/schema/ChannelListSchema';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ChannelList` from `../../database/schema/ChannelListSchema` is a new addition. Ensure that the path is correct and the exported module from the file is indeed `ChannelList`.

```diff
+ import ChannelList from '../../database/schema/ChannelListSchema';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->